### ISSUE TITLE
False negative on DELETE alias

### ISSUE THREAD

**Issue by Bretanark** [ErikEJ/SqlServer.Rules#352]
The following code reports a false negative "Object should have schema name (dbo)" on the first "sla" in
```
    DELETE sla
    FROM [$(Data)].dbo.SaleLifeAgent sla
    JOIN #MaxSaleLifeAgentOrder sl ON sl.SaleLifeId = sla.SaleLifeId
    WHERE sla.SortOrder > sl.MaxSortOrder
```
